lambdas/process-async-cri-credential/src/test/java/uk/gov/di/ipv/core/processasynccricredential/helpers/AsyncCriResponseHelperTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.di.ipv.core.processasynccricredential.helpers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.core.processasynccricredential.domain.ErrorAsyncCriResponse;
+import uk.gov.di.ipv.core.processasynccricredential.domain.SuccessAsyncCriResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.di.ipv.core.processasynccricredential.helpers.AsyncCriResponseHelper.DEFAULT_CREDENTIAL_ISSUER;
+import static uk.gov.di.ipv.core.processasynccricredential.helpers.AsyncCriResponseHelper.getAsyncResponseMessage;
+import static uk.gov.di.ipv.core.processasynccricredential.helpers.AsyncCriResponseHelper.isSuccessAsyncCriResponse;
+
+@ExtendWith(MockitoExtension.class)
+class AsyncCriResponseHelperTest {
+
+    private static final String TEST_USER = "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6";
+    private static final String TEST_OAUTH_STATE = "abcdabcdabcdabcd";
+
+    private static final String TEST_JWT = "jwt.as.string";
+
+    private static final String TEST_ERROR = "an error";
+    private static final String TEST_ERROR_DESCRIPTION = "a description of the error";
+    private static final String TEST_SUCCESS_ASYNC_RESPONSE_MESSAGE =
```

Comment:
This would be easier to read if it was done using String.format. Also, probably don't need the newlines as it's unlikely the target system will have new lines.